resolwe/flow/executors/__init__.py

```diff
@@ -112,63 +173,73 @@ def get_stdout(self):
         """Get process' standard output."""
         return self.stdout  # pylint: disable=no-member
 
-    def update_data_status(self, **kwargs):
-        """Update (PATCH) data object."""
-        data = Data.objects.get(pk=self.data_id)
-        for key, value in kwargs.items():
-            setattr(data, key, value)
+    def send_manager_command(self, cmd, expect_reply=True, **kwargs):
+        """Send a properly formatted command to the manager."""
+        assert EXECUTING
+
+        obj = {
+            'id': settings['Data_get']['id'],
+            'action': cmd,
```

Comment:
Why is this called `action` instead of `command`?

Reply:
Changed to use global constants, and changed to `command`.